### DISCUSSION TITLE
Can we disable/hide 'send a quick message' chat option in mgt-person-card

### DISCUSSION THREAD

**Discussion by BVSanjay** [microsoft-search/pnp-modern-search#3774] [General]
![image](https://github.com/microsoft-search/pnp-modern-search/assets/19494973/a7ad1a8d-4108-4c0a-975d-fd40c96c3a3d)
<img width="220" alt="mgt" src="https://github.com/microsoft-search/pnp-modern-search/assets/19494973/3c6fa7b6-80b8-4315-85b6-e17f475ae7e3">

**Comment by kasperbolarsen** [comment]
Not in this project as this component is used in this project as an out of the box component.

**Reply by BVSanjay** [reply]
I understand it is an out-of-the-box component but is there a property provided by the element to hide/disable it?

**Reply by BVSanjay** [reply]
@kasperbolarsen

**Reply by kasperbolarsen** [reply]
It is more likely that you will get an answer if you ask the question in a MGT forum. If so, please let us know here too.
#SharingIsCaring

**Reply by wobba** [reply]
You can check and ask at https://github.com/microsoftgraph/microsoft-graph-toolkit
